### ISSUE TITLE
Agenda for Aug 18 meeting

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/interop#110]
Here's the agenda for our meeting tomorrow:

- [Interop 2023 RFC](https://github.com/web-platform-tests/rfcs/blob/master/rfcs/interop_2023.md) now merged. Repo and team renamed, many old issues closed.
- Preparing for the public proposals process. Suggestion:
  - Process documented in the repo README
  - Issue templates for proposals
  - Call for proposals on our respective blogs and social media channels
- [Interop Team Charter RFC](https://github.com/web-platform-tests/rfcs/pull/120) updated, [actual charter](https://github.com/web-platform-tests/interop/pull/102) ready for review.
- Test change review
  - https://github.com/web-platform-tests/interop-2022/issues/107
  - https://github.com/web-platform-tests/interop-2022/issues/108
  - https://github.com/web-platform-tests/interop-2022/issues/109
- Investigation effort updates
  - Editing (@jgraham): https://github.com/web-platform-tests/interop-2022-editing
  - Pointer and Mouse Events (@flackr): https://github.com/web-platform-tests/interop-2022-pointer
    - [Interop 2022: Pointer Events Related Issues](https://docs.google.com/spreadsheets/d/1K3lnen0hP0T3z7lDmUnTtTrA-qWJ5JVTvkuvhV8AQhY/edit?usp=sharing) has 69 issues, 28 triaged. Expectation is that progress will be scored by checking the 4 status columns.
    - @flackr planning to schedule another meeting in a few weeks, suggesting that we finalize the list before then. @mustaqahmed will be looking over the crbug interop issues next week.
  - Viewport (@bramus): https://github.com/web-platform-tests/interop-2022-viewport
    - Bunch of test pages published at https://interop-2022-viewport.netlify.app/
    - Bunch of scenario test and its results published at https://goo.gle/interop-2022-viewport-testresults (_ongoing_)
    - CSSWG agreed to rename css-device-adapt to css-viewport, and Emilio will be editor. The venue for our suggestions would be there.
    - Explainers are being finished as we speak, and @bramus will publish them today
    - [Meeting 3](https://github.com/web-platform-tests/interop-2022-viewport/issues/4) scheduled for next week. Agenda TBD.

Previous meeting: https://github.com/web-platform-tests/interop-2022/issues/106

**Comment by foolip** [comment]
_Brief notes_

**Preparing for the public proposals process**

Philip: Anything missing?
Jen: We'll probably get more outside attention than last year. Materials need to be written from the pov from someone who doesn't have any background.
Philip: Should we focus this in the README?
Jen: In all the places!
Rachel: Happy to help wordsmith. But we'll still have lots of off-topic things and it's handy to have something to point to.
Philip: Let's have something ready for next week's meeting.

**Interop Team Charter**

Philip: I took a first stab at a scope, and included "all" SDOs and test suites. Test262 is for JS.
Brian: I think it's good to include, we have Intl and Temporal stuff coming up.
Sam: Current state of automated runners for Test262?
Philip: I think so, because there's test262.report, but I've never chased down the report files
Sam: Just wondering how hard it would be to get the results on a comparable form.
Brian: If we can lay out what we'd specifically like, I'd be happy to take an action item to find out if we can get X.
Jen: I think the charter language should be stronger, no "etc.". Other test suites can be considered but have to be able to be integrated by the end of the proposals period. If integrating Test262 is too hard.
Philip: Good point, we should figure this out before the proposals process starts.
Jen: Charter shouldn't keep changing, so it can say "right now WPT but open to other test suite that can integrate with WPT or wpt.fyi"

Brian: "primarily web-platform-tests. Other suites may be acceptable if they are <a href=....>integratable with our reporting process</a>"

Jen: The etc. for SDOs could also mean anything. Should list them all, and say we're open to including others in the future.
Brian: IETF should be on there I think
Sam: We could also define it with reference to the w3c [browser-specs](https://github.com/w3c/browser-specs) repo.
Jen: I think it shouldn't include WICG. "Web standards" not just specs. Does not include WICG and W3C Community Groups. "Rare exceptions can be may on a case by case basis."
Brian: Generally speaking I agree, but there are things that aren't standards track but have core agreement, like Web Speech API. If we identified a core set of tests, the fact that it hasn't run through a WG probably shouldn't block us.
Rachel: Defaulting to "on standards track" makes sense as a default. We could make exceptions.

**Test change review and Investigation effort updates** are FYIs, read the agenda :)

For next week:
- Update charter to be specific about which SDOs and test suites are in scope, while being open to extension/exceptions.
- Prepare PR for this repo's README and issue template to be used in the proposals process. Something we can also point to for any proposals that aren't in scope.
